server/api/api.go

```diff
@@ -101,6 +101,10 @@ func (a *API) RegisterRoutes(r *mux.Router) {
 
 	apiv1.HandleFunc("/workspaces", a.sessionRequired(a.handleGetUserWorkspaces)).Methods("GET")
 
+	// Insights APIs
+	apiv1.HandleFunc("/teams/{teamID}/boards/insights", a.sessionRequired(a.handleTeamBoardsInsights)).Methods("GET")
+	apiv1.HandleFunc("/users/me/{teamID}/boards/insights", a.sessionRequired(a.handleUserBoardsInsights)).Methods("GET")
```

Comment:
Should team be an optional parameter because there might be a point in future where these insights aren't team scoped?

Reply:
@mkraft, @chenilim, can you answer this?

Reply:
Going from required to optional isn't a breaking change to an API so I'm fine with us addressing that if it comes up in the future.